src/test/java/uk/gov/hmcts/reform/managecase/service/noc/ChallengeAnswerValidatorTest.java

```diff
@@ -210,6 +210,275 @@ void shouldErrorWhenFieldIsNullButAnswerProvided() {
         assertThat(exception.getErrorMessage(), is(ANSWERS_NOT_MATCH_LITIGANT.getErrorMessage()));
     }
 
+    @Test
+    void whenCaseFieldIsPresentAndAnswerIsNull_IgnoreNullFieldIsTrue() {
```

Comment:
Could you double-check the names of the new tests please - they don't seem to quite match up correctly e.g. this one is an error scenario but it's the following test which mentions an error (but is actually a success!)

Reply:
updated the test names to match with the functionality